ibis/backends/dask/execution/generic.py

```diff
@@ -43,6 +43,7 @@
     execute_node_nullif_series_scalar,
     execute_node_self_reference_dataframe,
     execute_null_if_zero_series,
+    execute_searched_case,
```

Comment:
What is this for?

Reply:
Please see the comments at the top of the PR. 

> This code also fixes #2711 by using da.select. This was only introduced in dask 2021.6.1, so I've ported the code into a shim. #2847 is a follow up issue to remove this.

Basically, working on windows I ran into a situation where it was easier to solve this in the same PR instead of making a work around on a work around.